### PR TITLE
Ug/style

### PR DESCRIPTION
**Pull request by artlee06**
Minor styling changes to the website
- Background color change to off white 
- headers remain orange but to our color code
- captions for images change to purple
- typography change to be mostly sans serif using open sans for body and montserrat for headers(as it was already imported)
- used prettier for asciidoc to make it more readable

### CODE REVIEW

File: docs/stylesheets/asciidoctor.css [hunk 1]
```diff
@@ -1,407 +1,1984 @@
...
+}
+h1,
+h2,
+h3,
+#toctitle,
+.sidebarblock > .content > .title,
+h4,
+h5,
+h6 {
+  font-family: "Montserrat", "Open Sans", sans-serif;
```
Comment (artlee06) [inline docs/stylesheets/asciidoctor.css:349]: header change here

File: docs/stylesheets/asciidoctor.css [hunk 2]
```diff
@@ -1,407 +1,1984 @@
...
+h1 small,
+h2 small,
+h3 small,
+#toctitle small,
+.sidebarblock > .content > .title small,
+h4 small,
+h5 small,
+h6 small {
+  font-size: 60%;
+  color: #453FE9;
```
Comment (artlee06) [inline docs/stylesheets/asciidoctor.css:367]: color change to purple here

File: docs/stylesheets/asciidoctor.css [hunk 3]
```diff
@@ -1,407 +1,1984 @@
...
+.verseblock > .title,
+.videoblock > .title,
+.dlist > .title,
+.olist > .title,
+.ulist > .title,
+.qlist > .title,
+.hdlist > .title {
+  text-rendering: optimizeLegibility;
+  text-align: left;
+  font-family: "Open Sans", "Arial", sans-serif;
```
Comment (artlee06) [inline docs/stylesheets/asciidoctor.css:1003]: font change to open sans here for table mainly

File: docs/stylesheets/gh-pages.css [hunk 4]
```diff
@@ -62,7 +62,7 @@ table.tableblock > .title,
 .ulist > .title,
 .qlist > .title,
 .hdlist > .title {
-    color: rgb(197, 90, 17);
+    color: #453FE9;
```
Comment (artlee06) [inline docs/stylesheets/gh-pages.css:65]: Color change for captions to purple

**Review by Permas-Teo — APPROVED** [review]
lgtm!

**Pull request merged**